github/event_types.go

```diff
@@ -1051,17 +1051,27 @@ type RepositoryDispatchEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// RepositoryImportEvent represents the activity related to a repository being imported to GitHub.
+//
+// GitHub API docs: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#repository_import
+type RepositoryImportEvent struct {
+	Status *string       `json:"status,omitempty"`
```

Comment:
Could you please add this comment?

```suggestion
	// Status represents the final state of the import. This can be one of "success", "cancelled", or "failure".
	Status *string       `json:"status,omitempty"`
```